src/api/Metadata.js

```diff
@@ -226,14 +226,17 @@ class Metadata extends File {
      * Gets metadata instances for a Box file
      *
      * @param {string} id - file id
+     * @param {boolean} isMetadataRedesign - feature flag
      * @return {Object} array of metadata instances
      */
-    async getInstances(id: string): Promise<Array<MetadataInstanceV2>> {
+    async getInstances(id: string, isMetadataRedesign: boolean = false): Promise<Array<MetadataInstanceV2>> {
         this.errorCode = ERROR_CODE_FETCH_METADATA;
+        const baseUrl = this.getMetadataUrl(id);
+        const url = isMetadataRedesign ? `${baseUrl}?view=hydrated` : baseUrl;
```

Comment:
nit: there are some patterns in this file which use a separate function to build the URLs

Reply:
We have some taxonomy node related API code in progress. I'll look to use some of these patterns as part of that work